doc/conf.py

```diff
@@ -381,7 +381,12 @@ def js_tag_with_cache_busting(js):
     "logo": {"link": "index",
              "image_light": "images/logo2.svg",
              "image_dark": "images/logo_dark.svg"},
-    "navbar_end": ["theme-switcher", "version-switcher", "mpl_icon_links"]
+    "navbar_end": ["theme-switcher", "version-switcher", "mpl_icon_links"],
+    "page_sidebar_items": [
+        "page-toc.html",
+        "searchbox.html",
```

Comment:
We don't want the *searchbox*

```suggestion
```

Unsure if we need *edit-this-page*. At least in the the CI-built docs it does not seem to have an effect.

Reply:
Sorry, I didn't carefully check the built docs - fixed.  
Although I don't quite understand why *searchbox* and *edit-this-page* are not shown in the docs: `mpl-sphinx-theme` inherits from `pydata-sphinx-theme` which by default [includes](https://github.com/pydata/pydata-sphinx-theme/blob/02926a869056321f4aa86885d23400405db1037a/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/theme.conf#L37) both *searchbox* and *edit-this-page* in the sidebar. Where is this excluded/overridden in [`theme.conf`](https://github.com/matplotlib/mpl-sphinx-theme/blob/main/mpl_sphinx_theme/theme.conf) or [`conf.py`](https://github.com/matplotlib/matplotlib/blob/main/doc/conf.py)?